common/lib/xmodule/xmodule/js/spec/problem/edit_spec.coffee

```diff
@@ -508,21 +509,21 @@ describe 'MarkdownEditingDescriptor', ->
         Test Explanation.
         [Explanation]
         """)
-      expect(data).toEqual("""<problem>
-    <p>Who lead the civil right movement in the United States of America?</p>
-    <stringresponse answer="w*.?s*Luther Kings*.*" type="ci regexp" >
-      <textline label="Who lead the civil right movement in the United States of America?" size="20"/>
-    </stringresponse>
+      expect(data).toXMLEqual("""<problem>
+        <stringresponse answer="w*.?s*Luther Kings*.*" type="ci regexp">
+          <label >Who lead the civil right movement in the United States of America?</label>
+        <textline label="Who lead the civil right movement in the United States of America?" size="20"/>
```

Comment:
Given that we now have a label tag being created, should the textline element be removed? I'm not clear why there is the duplication.